package.json

```diff
@@ -83,10 +66,33 @@
     "sass": "^1.26.12",
     "striptags": "^3.1.1",
     "styled-components": "^5.2.0",
-    "stylelint": "^13.8.0",
-    "stylelint-config-recess-order": "^2.3.0",
-    "stylelint-scss": "^3.18.0",
     "swr": "^0.5.6",
     "use-resize-observer": "^6.1.0"
+  },
+  "devDependencies": {
+    "@babel/eslint-parser": "^7.11.5",
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@babel/plugin-transform-react-jsx": "^7.10.4",
+    "eslint": "^7.14.0",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-config-standard": "^16.0.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "json-to-scss": "^1.6.2",
+    "npm-run-all": "^4.1.5",
+    "postcss-normalize": "^10.0.1",
```

Comment:
`postcss-normalize` updated from `9.0.0` to `10.0.1` to resolve a dependency issue after `sanitize.css` was updated when the `yarn.lock` file was refreshed